src/_gettsim/einkommensteuer/einkünfte/einkünfte.py

```diff
@@ -0,0 +1,50 @@
+"""Einkünfte according to §§ 13-24 EStG."""
+
+from ttsim import policy_function
+
+
+@policy_function(end_date="2008-12-31", leaf_name="gesamtbetrag_der_einkünfte_y")
+def gesamtbetrag_der_einkünfte_y_mit_kapiteleinkünften(
```

Comment:
Let's stick to the order from the [law](https://www.gesetze-im-internet.de/estg/__2.html):

1.    Einkünfte aus Land- und Forstwirtschaft,
2.    Einkünfte aus Gewerbebetrieb,
3.    Einkünfte aus selbständiger Arbeit,
4.    Einkünfte aus nichtselbständiger Arbeit,
5.    Einkünfte aus Kapitalvermögen,
6.    Einkünfte aus Vermietung und Verpachtung,
7.    sonstige Einkünfte im Sinne des § 22,